en/heroku/README.md

```diff
@@ -204,7 +204,7 @@ We can now visit the app in our browser with `heroku open`.
 
 This will open a url like [https://djangogirlsblog.herokuapp.com/]() in your browser, and at the moment you will probably see an error page.
 
-The error you saw was because we when we deployed to Heroku, we created a new database and it's empty. We need to run the `migrate` and `createsuperuser` commands, just like we did on PythonAnywhere.  This time, they come via a special command-line on our own computer, `heroku run`:
+The error you saw was because when we deployed to Heroku, we created a new database and it's empty. We need to run the `migrate` and `createsuperuser` commands, just like we did on PythonAnywhere.  This time, they come via a special command-line on our own computer, `heroku run`:
```

Comment:
Good catch, but unrelated to the Farsi translation. Can you file this change as a separate Pull Request? Thanks!

Reply:
OK